src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt

```diff
@@ -81,6 +81,8 @@ fun insertTitleAndResizableTextIntoPanelColumns(
 
 fun snykCodeAvailabilityPostfix(): String = when {
     !isSnykCodeAvailable(pluginSettings().customEndpointUrl) -> " (disabled for endpoint)"
+    (pluginSettings().sastOnServerEnabled ?: true && pluginSettings().localCodeEngineEnabled ?: true) ->
+        " (disabled due to local code engine)"
```

Comment:
This should be `Local Code Engine` (capitalization)

Reply:
👍, fixed with f81b3396a7de38ace4ab5f5f57f886b714dd0f4b